doc/source/modifiers.rst

```diff
@@ -52,6 +52,57 @@ on those modifiers can be found in the linked API documentation.
       - Modifies solar channels to filter out the visual impact of rayleigh
         scattering.
 
+A complete list can be found in the ``etc/composites`` source code and
+in the :mod:`~satpy.modifiers` module documentation.
+
+Parallax correction
+-------------------
+
+.. warning::
+
+    The Satpy parallax correction is experimental and subject to change.
+
+Since early 2022, Satpy has included a
```

Comment:
Maybe it should be a version number here instead?

Reply:
Fixed in 1d953a80.